src/shared/utils/extractUploads.js

```diff
@@ -1,86 +0,0 @@
-import countBy from 'lodash/countBy'
-import groupBy from 'lodash/groupBy'
-
-import { UploadStateEnum, UploadTypeEnum } from 'shared/utils/commit'
-
-function humanReadableOverview(state, count) {
-  if (state === UploadStateEnum.error) return 'errored'
-  if (state === UploadStateEnum.processed) return 'successful'
-  if (state === UploadStateEnum.complete) return 'carried forward'
-  if (state === UploadStateEnum.started) return 'started'
-}
-
-function deleteDuplicateCFFUploads({ uploads }) {
-  const nonCFFFlags = []
-  const duplicateUploads = uploads && [...uploads]
-
-  // Get all the flags from "uploaded" uploads
-  duplicateUploads?.forEach((upload) => {
-    if (
-      upload?.uploadType !== UploadTypeEnum.CARRIED_FORWARD &&
-      upload?.flags
-    ) {
-      nonCFFFlags.push(...upload.flags)
```

Comment:
converted this to a set from an array so we can use .has() instead of .includes()